tools/querytee/proxy_metrics.go

```diff
@@ -43,6 +44,12 @@ func NewProxyMetrics(registerer prometheus.Registerer) *ProxyMetrics {
 			Name:      "responses_compared_total",
 			Help:      "Total number of responses compared per route name by result.",
 		}, []string{"route", "result"}),
+		relativeDuration: promauto.With(registerer).NewHistogramVec(prometheus.HistogramOpts{
+			Namespace: queryTeeMetricsNamespace,
+			Name:      "backend_request_relative_duration_seconds",
+			Help:      "Time (in seconds) of preferred backend less secondary backend.",
+			Buckets:   []float64{-100, -50, -25, -10, -5, -4, -3, -2, -1.5, -1, -.75, -.5, -.25, -.1, -.05, -.025, -.01, -.005, .005, .01, .025, .05, .1, .25, .5, 0.75, 1, 1.5, 2, 3, 4, 5, 10, 25, 50, 100},
+		}, []string{"method", "route"}),
```

Comment:
This might be a good candidate for a native histogram.

Reply:
I don't think negatives can be used in native histograms (correct me if I'm wrong). In which case we'd need two metrics: one for when the primary is more performant, and one for the secondary. WDYT?

Reply:
Ah, then I am mistaken. I will try this then, thanks.